cargo-dist/src/backend/ci/github.rs

```diff
@@ -42,6 +42,10 @@ pub struct GithubCiInfo {
     pub install_dist_sh: String,
     /// expression to use for installing dist via powershell script
     pub install_dist_ps1: String,
+    /// expression to use for installing cargo-auditable via shell script
+    pub install_cargo_auditable_sh: String,
+    /// expression to use for installing cargo-auditable via powershell script
+    pub install_cargo_auditable_ps1: String,
```

Comment:
I believe we can drop these two fields since we're exclusively installing it via the GitHub matrix now?